src/main/java/mate/academy/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -35,7 +35,7 @@ public MovieSession add(MovieSession movieSession) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new DataProcessingException("Can't insert movie session" + movieSession, e);
+            throw new DataProcessingException("Can't insert movie session: " + movieSession, e);
```

Comment:
The exception handling here is appropriate, as it rolls back the transaction if an error occurs and wraps the exception in a `DataProcessingException`. This provides a clear message about the failure and includes the original exception for debugging.